<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MultiSpeakerClass</class>
 <widget class="QMainWindow" name="MultiSpeakerClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>739</width>
    <height>656</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MultiSpeaker</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_7">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QSplitter" name="MainSplitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="TitleHeaderWidget" name="TopologyHeader" native="true"/>
        </item>
        <item>
         <widget class="HostView" name="TheHostView"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="TimelineContainerWidget" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="TimelineHeaderWidget" name="TimelineHeader" native="true">
         </widget>
        </item>
        <item>
         <widget class="QSplitter" name="TimelineSplitter">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <widget class="QWidget" name="layoutWidget_1">
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <property name="spacing">
             <number>0</number>
            </property>
            <item>
             <widget class="TitleHeaderWidget" name="MethodEventsHeader" native="true">
              <layout class="QHBoxLayout" name="horizontalLayout">
               <property name="spacing">
                <number>0</number>
               </property>
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QListView" name="TimelineEventListView"/>
            </item>
           </layout>
          </widget>
          <widget class="TimelineView" name="TheTimelineView">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>TimelineView</class>
   <extends>QGraphicsView</extends>
   <header>TimelineView.h</header>
  </customwidget>
  <customwidget>
   <class>HostView</class>
   <extends>QGraphicsView</extends>
   <header>HostView.h</header>
  </customwidget>
  <customwidget>
   <class>TitleHeaderWidget</class>
   <extends>QWidget</extends>
   <header>TitleHeaderWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TimelineHeaderWidget</class>
   <extends>QWidget</extends>
   <header>TimelineHeaderWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="MultiSpeaker.qrc"/>
 </resources>
 <connections/>
</ui>
